sa/proto/sa.proto

```diff
@@ -429,3 +431,13 @@ message PauseIdentifiersResponse {
   int64 paused = 1;
   int64 repaused = 2;
 }
+
+message UpdateRegistrationContactRequest {
+  int64 registrationID = 1;
+  repeated string contact = 2;
+}
+
+message UpdateRegistrationKeyRequest {
+  int64 registrationID = 1;
+  bytes jwk = 2;
+}
```

Comment:
Make sure there's a newline at the end of the file. Your editor should be able to be configured to ensure this is always true.